dandiapi/api/models/zarr.py

```diff
@@ -143,9 +140,40 @@ def to_checksum(self) -> ZarrChecksum:
         return ZarrChecksum(path=self.path, md5=self.etag)
 
 
-class ZarrArchive(TimeStampedModel):
+class ZarrUploadFile(BaseZarrUploadFile):
+    class Meta:
+        get_latest_by = 'modified'
+
+    blob = models.FileField(blank=True, storage=get_storage)
+    """The fully qualified S3 object key"""
+
+    zarr_archive: 'ZarrArchive' = models.ForeignKey(
+        'ZarrArchive',
+        related_name='active_uploads',
+        on_delete=models.CASCADE,
+    )
+
+
+class EmbargoedZarrUploadFile(BaseZarrUploadFile):
+    class Meta:
+        get_latest_by = 'modified'
```

Comment:
Can this not be placed in `BaseZarrUploadFile`, since it's the same across this and `ZarrUploadFile`?

Reply:
You apparently can, neat.